include/odp/api/abi-default/atomic.h

```diff
@@ -38,6 +38,14 @@ struct odp_atomic_u64_s {
 	uint64_t v; /**< Actual storage for the atomic variable */
 } ODP_ALIGNED(sizeof(uint64_t)); /* Enforce alignment! */
 
+/**
+ * @internal
+ * Atomic 128-bit unsigned integer
+ */
+struct odp_atomic_u128_s {
+	odp_u128_t v; /**< Actual storage for the atomic variable */
+} ODP_ALIGNED(sizeof(odp_u128_t)); /* Enforce alignment! */
```

Comment:
According to GCC documentation align attribute should be after "struct" keyword. This should be corrected also for other atomic type structs. Could do that also for u64/u32 (as a separate patch).

https://gcc.gnu.org/onlinedocs/gcc/Common-Type-Attributes.html

Reply:
ok.